### ISSUE TITLE
[IDEA] New repository

### ISSUE THREAD

**Issue by silvioprog** [silvioprog/brookframework#168]
Dear all,

it would be nice to have two new repositories at [Risoflora organization](https://github.com/risoflora) to host Brook project, something like:

1. https://github.com/risoflora/brookframework (the current [Sagui-based version](https://github.com/silvioprog/brookframework), providing support for Free Pascal & Delphi)
2. https://github.com/risoflora/brookfreepascal (the Free Pascal-only version, based on Brook 3 and 4 + all patches applied, and could receive new commits and/or release new versions)

Thus, any Risoflora developer could push commits to both (in case of large changes, only after previous discussion via issue or mailing list), and other users could send pull-requests. The [old repository](https://github.com/silvioprog/brookframework) would be archived and redirected to the new one.

If you have more ideas and/or suggestions, please share! :smiley:

**Comment by silvioprog** [comment]
Done: https://github.com/risoflora/brookframework.
